.github/checkgroup.yml

```diff
@@ -108,12 +118,10 @@ subprojects:
 
   - id: "lightning_app"
     paths:
-      - ".azure/app-cloud-e2e.yml"
       - "requirements/app/**"
       - "src/lightning_app/**"
       - "tests/tests_app/**"
       - "tests/tests_app_examples/**"
-      - "tests/tests_clusters/**"
```

Comment:
This test does not run at all: https://github.com/Lightning-AI/lightning/pull/13835#discussion_r950598910